### PR TITLE
update instructions for generic-worker dev

### PR DESCRIPTION
**Pull request by djmitche**
This reflects what I had to do to get `./build.sh -t` passing.

It looks like some of these `go get` lines are in `build.sh`, but not the `go install` lines.  I recall that in a recent version (go1.18?) `go get` stopped installing binaries.  Maybe it makes sense to have `build.sh` run `go install` for _all_ of these, instead of this solution?

### CODE REVIEW

File: workers/generic-worker/README.md [hunk 1]
```diff
@@ -281,8 +281,19 @@ See the [![GoDoc](https://pkg.go.dev/github.com/taskcluster/taskcluster/v42/work
 Set up to build Taskcluster in general.
 See [development process](../../dev-docs/development-process.md).
 
-* Run `go get github.com/taskcluster/livelog`
- Run `go get github.com/taskcluster/taskcluster/v30/tools/taskcluster-proxy`
+* Run
+
+  ```shell
+  go get github.com/taskcluster/livelog`
+  go install github.com/taskcluster/livelog`
```
Comment (matt-boris) [inline workers/generic-worker/README.md:288]:
```suggestion
  go get github.com/taskcluster/livelog
  go install github.com/taskcluster/livelog
```

**Review by matt-boris — CHANGES_REQUESTED** [review]
Can confirm these steps work in getting `./build.sh -t` to run successfully.

Just requested changes for a couple stray ` + a missing changelog, thanks!

File: workers/generic-worker/README.md [hunk 2]
```diff
@@ -281,8 +281,19 @@ See the [![GoDoc](https://pkg.go.dev/github.com/taskcluster/taskcluster/v42/work
...
+  go get github.com/taskcluster/taskcluster/v30/tools/taskcluster-proxy
+  go install github.com/taskcluster/taskcluster/v30/tools/taskcluster-proxy
```
Comment (petemoore) [inline workers/generic-worker/README.md:290]: v30?

**Review by petemoore — CHANGES_REQUESTED** [review]
Thanks for updating the docs.

I think you can avoid needing the `go get` and the `go install` by installing a specific version, e.g.

```
go install github.com/taskcluster/taskcluster/v44/tools/taskcluster-proxy@v44.9.2
```

If we want this to stay up-to-date, we should probably punch in the version numbers at release time via regexp. What do you think?

**Comment by petemoore** [comment]
> Maybe it makes sense to have `build.sh` run `go install` for _all_ of these, instead of this solution?

That's a fair point - I'm also happy for us to do it that way too, and the instructions be clear that you just run `build.sh`.

File: workers/generic-worker/README.md (hunk 2 above)
Comment (djmitche) [reply workers/generic-worker/README.md:290]: That was just from copy-paste :)

**Comment by djmitche** [comment]
It appears that this fixes #5774!

**Review by petemoore — DISMISSED** [review]

**Comment by petemoore** [comment]
@djmitche Can you add a silent changelog?

**Comment by djmitche** [comment]
> @djmitche Can you add a silent changelog?

Done

**Review by lotas — APPROVED** [review]

**Review by petemoore — APPROVED** [review]
Many thanks!

**Comment by lotas** [comment]
Somehow I merged it locally after doing rebase, and it didn't automatically closed this PR, sorry.

Here's the merge commit https://github.com/taskcluster/taskcluster/commit/ed6c810036c94393ecd38e49e578d88042b23d09  (all commits preserved in history also)

**Pull request closed without merging**